about_processes.exs

```diff
@@ -0,0 +1,37 @@
+defmodule AboutProcesses do
+  use Koans
+
+  think "You are a process with a process ID" do
+    pid_name = inspect(self)
+
+    assert_? pid_name =~ ~r/#PID<\d+\.\d+\.\d+>/
```

Comment:
Idk, I feel like that would put emphasis on how to write a regex rather than "see, this is a thing called a PID, process _ID_". I started by looking for something like `is_pid(value)`, but I didn't find anything.

I wonder if we could do something with Process.info/1 or Process.whereis/1

```elixir
iex(5)> Process.info(pid)
[current_function: {:prim_eval, :receive, 2},
 initial_call: {:erlang, :apply, 2}, status: :waiting, message_queue_len: 0,
 messages: [], links: [], dictionary: [], trap_exit: false,
 error_handler: :error_handler, priority: :normal, group_leader: #PID<0.26.0>,
 total_heap_size: 233, heap_size: 233, stack_size: 9, reductions: 17,
 garbage_collection: [min_bin_vheap_size: 46422, min_heap_size: 233,
  fullsweep_after: 65535, minor_gcs: 0], suspending: []]
```